boilerplates/vue/files/layouts/LayoutDefault.vue

```diff
@@ -17,6 +17,10 @@ import Content from "../components/Content.vue";
 import Link from "../components/Link.vue";
 import Logo from "../components/Logo.vue";
 import Sidebar from "../components/Sidebar.vue";
+/*{ @if (it.BATI.has("sentry")) }*/
+import { sentryBrowserConfig } from "../sentry.browser.config";
+sentryBrowserConfig();
+/*{ /if }*/
```

Comment:
Prefer this syntax when you can. Unused imports are automatically removed.

```suggestion
import { sentryBrowserConfig } from "../sentry.browser.config";
//# BATI.has("sentry")
sentryBrowserConfig();
```